webpack.config.js

```diff
@@ -0,0 +1,47 @@
+const entryPointNames = [
```

Comment:
There's a `getAllPackages` and `getPackageNameFromFile` functions in the `scripts/build.js` file. Could be extracted into a helper file.